File: src/SextantTG/SextantTG.ActiveRecord/Tour.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SextantTG.ActiveRecord
{
    /// <summary>
    /// 旅行实体类
    /// </summary>
    public class Tour : BaseAR
    {
        /// <summary>
        /// 旅行ID
        /// </summary>
        public string TourId { get; set; }

        /// <summary>
        /// 旅行名称
        /// </summary>
        public string TourName { get; set; }

        /// <summary>
        /// 相关的用户ID
        /// </summary>
        public string UserId { get; set; }

        /// <summary>
        /// 起始日期
        /// </summary>
        public DateTime? BeginDate { get; set; }

        /// <summary>
        /// 截止日期
        /// </summary>
        public DateTime? EndDate { get; set; }

        /// <summary>
        /// 金额
        /// </summary>
        public int? Cost { get; set; }

        /// <summary>
        /// 旅行状态
        /// </summary>
        public int? Status { get; set; }

        /// <summary>
        /// 创建时间
        /// </summary>
        public DateTime? CreatingTime { get; set; }

        /// <summary>
        /// 备注
        /// </summary>
        public string Memos { get; set; }

        public override string ToString()
        {
            return string.Format("ID:{0}, 名称:{1}, 时间:{2}-{3}, 状态:{4}, 金额:{5}", TourId, TourName, BeginDate, EndDate, Status, Cost);
        }
    }
}

File: src/App/src/com/ss/stg/model/Tour.java
package com.ss.stg.model;

import java.io.Serializable;
import java.util.Date;

/**
 * 旅行
 * 
 * @author ggxx
 * 
 */
public class Tour implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = 7235312371778254249L;
	private String tourId;
	private String tourName;
	private String userId;
	private Date beginDate;
	private Date endDate;
	private int cost;
	private int status;
	private Date creatingTime;
	private String memos;

	public Tour() {
	}

	public String getTourId() {
		return tourId;
	}

	public void setTourId(String tourId) {
		this.tourId = tourId;
	}

	public String getTourName() {
		return tourName;
	}

	public void setTourName(String tourName) {
		this.tourName = tourName;
	}

	public Date getBeginDate() {
		return beginDate;
	}

	public void setBeginDate(Date beginDate) {
		this.beginDate = beginDate;
	}

	public Date getEndDate() {
		return endDate;
	}

	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}

	public int getCost() {
		return cost;
	}

	public void setCost(int cost) {
		this.cost = cost;
	}

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId;
	}

	public int getStatus() {
		return status;
	}

	public void setStatus(int status) {
		this.status = status;
	}

	public Date getCreatingTime() {
		return creatingTime;
	}

	public void setCreatingTime(Date creatingTime) {
		this.creatingTime = creatingTime;
	}

	public String getMemos() {
		return memos;
	}

	public void setMemos(String memos) {
		this.memos = memos;
	}
}
